src/docs/product/releases/health/index.mdx

```diff
@@ -26,21 +26,29 @@ A session ends with the closing of the application or with the application being
 
 Sessions are submitted to Sentry so you can track the usage and adoption of your application. When a user of your application experiences a crash, error, or abnormal exit, the session will be flagged accordingly, and Sentry calculates derived metrics. The metrics include data such as the number of users that didn't experience a crash in the specified time range.
 
-### Active Users
+### Active Sessions / Users
 
-Users who started the application at least once in the last 24 hours.
+Active Sessions are sessions that started in the application in the last 24 hours.
+
+Active Users are users who started the application at least once in the last 24 hours.
+
+To sort releases by active sessions, select Display: Sessions and to sort by active users select Display: Users.
+
+![Sort By: Active Sessions / Users dropdown](release_active.png)
 
 ### Crash
 
 The app had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue in [sentry.io](https://sentry.io) that captures this event, and errors that did not cause the end of the application should not be included. To search for them in **Discover** or on the **Issues** page, filter by `error.unhandled:true `. The number of unhandled events is not expected to match the number of crashed sessions because sessions are not subject to [Inbound Filters](/product/data-management-settings/filtering/) or [Sampling](/platform-redirect/?next=/configuration/sampling/).
 
-### Crash Free User
+### Crash Free Sessions / Users
+
+Crash Free Sessions is the percentage of sessions in the specified time range not ended by a crash of the application.
```

Comment:
To get around the issue of "crash free sessions" being plural followed by a singular verb, I've put quotations around it and kept it capitalized to denote that we're referring to the (singular) UI element. 

```suggestion
"Crash Free Sessions" is the percentage of sessions in the specified time range not ended by a crash of the application.
```